tasks/copy_from_archive/copy_from_archive.py

```diff
@@ -132,7 +132,7 @@ def task(
                 a_file[FILE_SUCCESS_KEY] for a_file in files
             ):  # Check for early completion
                 break
-            LOGGER.warn(
+            LOGGER.warning(
```

Comment:
looks like `LOGGER.warn` works fine. 
Any reason to change it to `LOGGER.warning`?

Reply:
In the logger.logging library, `warn` is deprectated.